samples/TestApp.PCL/TokenBroker.cs

```diff
@@ -75,10 +73,14 @@ public MobileAppSts()
 
     public class TokenBroker
     {
-        private AuthenticationContext context;
 
-        public Sts Sts = new AadSts();
+        private Sts sts = new AadSts();
```

Comment:
Minor changes in this file to fix some code analysis issues (all low risk, all in test code)